code/modules/holodeck/holodeck_map_templates.dm

```diff
@@ -163,3 +175,17 @@
 	description = "benis"
 	mappath = "_maps/templates/holodeck_refuelingstation.dmm"
 	restricted = TRUE
+
+/datum/map_template/holodeck/asylum
+	name = "Holodeck - Asylum"
+	template_id = "holodeck_asylum"
+	description = "benis"
+	mappath = "_maps/templates/holodeck_asylum.dmm"
+	restricted = TRUE
+
+/datum/map_template/holodeck/clownworld
+	name = "Holodeck - Clown World"
+	template_id = "holodeck_clownworld"
+	description = "benis"
```

Comment:
change

Reply:
I don't even know why it's there. It is never used/refferenced... Ever.